requirements/release.txt

```diff
@@ -3,10 +3,12 @@ celery[redis]>=4.3.0
 coloredlogs
 cryptography
 importlib_resources; python_version < '3.7'
+maestrowf
```

Comment:
this really needs to be maestrowf>=1.1.5, which doesn't exist yet. so for now, we need to keep maestro install via GitHub develop

Reply:
Addressed. Added in commented-out `maestrowf>=1.1.5` to remind us to use it once it exists.